src/core/packages/http/server/src/router/request.ts

```diff
@@ -214,3 +214,8 @@ export interface KibanaRequest<
    */
   readonly body: Body;
 }
+
+/**
+ * @remark Convenience type, use when the concrete values of P, Q, B and route method do not matter.
```

Comment:
WDYT about adding a typical example of when concrete values don't matter? It might help prevent unintended use of `AnyKibanaRequest`